New log file opened at Mon Oct 05 13:02:53 CEST 2015 by 1636@PC-14874
Taking next connection to: 59403
New client attempting to connect on port: 59404-59403
  Address = /139.63.20.189:59403
This client on port 59404 has uuid df870866-7df7-4e1a-bfa5-f6dd8378582c configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-df870866-7df7-4e1a-bfa5-f6dd8378582c
Taking next connection to: 59403
New client attempting to connect on port: 59405-59403
  Address = /139.63.20.189:59403
Handshake exception on socket: 59405
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59403
New client attempting to connect on port: 59406-59403
  Address = /139.63.20.189:59403
This client on port 59406 has uuid afb29b31-22ec-437a-882b-55b3322b7edb configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-df870866-7df7-4e1a-bfa5-f6dd8378582c, activator-afb29b31-22ec-437a-882b-55b3322b7edb
Taking next connection to: 59403
New client attempting to connect on port: 59407-59403
  Address = /139.63.20.189:59403
Handshake exception on socket: 59407
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59403
New client attempting to connect on port: 59408-59403
  Address = /139.63.20.189:59403
This client on port 59408 has uuid f39df6b4-7e34-4099-a625-86d24adc7cfb configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-df870866-7df7-4e1a-bfa5-f6dd8378582c, activator-afb29b31-22ec-437a-882b-55b3322b7edb, play-fork-run-f39df6b4-7e34-4099-a625-86d24adc7cfb
Taking next connection to: 59403
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59403
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59403
New client attempting to connect on port: 59532-59403
  Address = /139.63.20.189:59403
Handshake exception on socket: 59532
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59403
New client attempting to connect on port: 59533-59403
  Address = /139.63.20.189:59403
This client on port 59533 has uuid c2db9a6a-6f88-4dcc-bc61-1d8cede9c818 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-afb29b31-22ec-437a-882b-55b3322b7edb, play-fork-run-c2db9a6a-6f88-4dcc-bc61-1d8cede9c818
Taking next connection to: 59403
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59403
